packages/renderer/src/lib/container/ContainerDetailsLogsClear.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { faEraser } from '@fortawesome/free-solid-svg-icons';
+import type { Terminal } from '@xterm/xterm';
+import Fa from 'svelte-fa';
+
+const { terminal }: { terminal: Terminal } = $props();
+
+function clear(): void {
+  terminal.clear();
+}
+</script>
+
+<div class="absolute top-0 right-0 px-1 z-50 m-1 opacity-50 space-x-1">
+  <button title="Clear logs" onclick={clear}>
+    <Fa
+      class="cursor-pointer rounded-full bg-[var(--pd-button-disabled)] min-h-8 w-8 p-1.5"
```

Comment:
Missing a hover effect.

The icon isn't as bright as the actions on top, but if I remove the opacity it is too bright. I think it's missing a text- and regular opacity would help with the previous comment.